### DISCUSSION TITLE
enableReactComponents doesnt work in codesandboxed vite react app

### DISCUSSION THREAD

**Discussion by yousif-khalil-abdulkarim** [LegendApp/legend-state#206] [Q&A] [answered]
enableReactComponents doesnt work in codesandboxed vite react app. [code sandbox link](https://codesandbox.io/p/sandbox/legend-state-test-hwtx7h?file=%2Fsrc%2FApp.tsx%3A24%2C14).

Please tell me if I have missed something

**Comment by jmeistrich** [comment, marked as answer]
`enableReactComponents` is for enabling the Reactive components like `Reactive.div`: https://legendapp.com/open-source/state/fine-grained-reactivity/#reactive-components

But it does not enable rendering observables directly. For that you'd want to use `Memo`: https://legendapp.com/open-source/state/fine-grained-reactivity/#render-an-observableselector-directly
